moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -0,0 +1,837 @@
+/*********************************************************************
```

Comment:
Are there any changes in this file beyond the class name? It would be hard to review if you have changed anything